var/spack/repos/builtin/packages/py-pennylane/package.py

```diff
@@ -10,12 +10,24 @@
 class PyPennylane(PythonPackage):
     """PennyLane is a Python quantum machine learning library by Xanadu Inc."""
 
-    homepage = "https://github.com/XanaduAI/pennylane"
+    homepage = "https://docs.pennylane.ai/"
+    git = "https://github.com/PennyLaneAI/pennylane.git"
     pypi = "PennyLane/PennyLane-0.28.0.tar.gz"
 
-    maintainers("marcodelapierre")
-
-    version("0.28.0", sha256="2a6100c00277c1eb59eab6856cdad7b1237e9d1fbda98b1e15020bd5a64b10a8")
+    maintainers("mlxd", "AmintorDusko", "marcodelapierre")
+
+    version("master", branch="master")
+    version("0.29.1", sha256="c5d662994b741afa69e4fdadc79a1b75840275138a8b7e0cfc5fd64b66a12eef")
+    version(
+        "0.29.0",
+        sha256="055fc22a2624cceddbb8bd25d7eda14d2dd7e0afa08ea8555c23bc75eff007a5",
+        deprecated=True,
```

Comment:
Why add 0.29.0 if it's deprecated?

Reply:
We didn't want a discontinuity in the history of versions listed, but I agree this isn't sensible since there's no historical availability of the package. Will remove it.